depsync/syncer.go

```diff
@@ -0,0 +1,149 @@
+package depsync
+
+import (
```

Comment:
Not sure what IDE you're using, but the imports need to be sorted - stdlib imports first, then a space, then other imports. `goimports` will do that for you.